rh-summit-2018/module-01.adoc

```diff
@@ -251,15 +251,12 @@ To deploy Grafana on OpenShift, the following commands should be executed:
 $ oc create -f https://raw.githubusercontent.com/strimzi/strimzi/master/metrics/examples/grafana/kubernetes.yaml
 
 For accessing the dashboard, you can use the `port-forward` command for forwarding traffic from the Grafana pod to the host.
-For example, you can access the Grafana UI by running :
+For example, you can access the Grafana UI after running:
 
 [source,sh]
-$ oc port-forward grafana-1-fbl7s 3000:3000
+$ oc expose svc/grafana
```

Comment:
now that we are exposing the Grafana dashboard through a Route, should we say where the attendee can find the link to that Route for opening the UI ?

Reply:
We should point them to the Routes tab in the UI, or just how to get it from the CLI.

Reply:
As a side note, we should do the same in module 2 (and wherever we're having routes).